descriptor.md

```diff
@@ -28,13 +28,14 @@ The following describe the primary set of properties that make up a _Descriptor_
   This property exists so that a client will have an expected size for the content before validating.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-### Reserved
+- **`urls`** *array*
 
-The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
+  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
+  For an ordinary layer, this property is empty and the layer contents can be retrieved directly from the registry.
```

Comment:
Can we drop this line?  I have a few concerns with it:

1. This is the descriptor spec and “layer” (i.e. application/vnd.oci.image.serialization.rootfs.tar.gzip) is a higher level idea.  You could rename “layer” → “blob”, but…
2. Who knows if you can get the content from “the registry”?  What registry are we talking about anyway?  The descriptor spec doesn't currently give you a way to say “and you can fetch this blob from this registry” (that's sort of what `urls` is for).

The previous line clearly declares `urls` as OPTIONAL, so I think it will be pretty clear to implementers that in its absence, they should be asking for the blob in any CAS engines they know about, as configured by the caller.  I don't think we gain any additional clarity by waving our hands about “the registry” here.